src/iterator.rs

```diff
@@ -154,13 +149,38 @@ pub trait PointCloud: Sync {
     type Id: ToString + Send + Copy;
     fn nodes_in_location(&self, location: &PointLocation) -> Vec<Self::Id>;
     fn encoding_for_node(&self, id: Self::Id) -> Encoding;
-    fn points_in_node<'a>(
-        &'a self,
-        query: &'a PointQuery,
+    /// Return all points in the selected node.
```

Comment:
shall the Point Cloud trait be moved also somewhere else while reformatting this part?

Reply:
I'd keep the PR small for now, but I can do it soon :)